api/mapping/views.py

```diff
@@ -161,45 +191,45 @@ def get_context_data(self, **kwargs):
             scan_report_field = None
             scan_report_value = None
 
-        context.update({
-            'scan_report': scan_report,
-            'scan_report_table': scan_report_table,
-            'scan_report_field': scan_report_field,
-            'scan_report_value': scan_report_value,
-        })
+        context.update(
+            {
+                "scan_report": scan_report,
+                "scan_report_table": scan_report_table,
+                "scan_report_field": scan_report_field,
+                "scan_report_value": scan_report_value,
+            }
+        )
 
         return context
 
 
 @method_decorator(login_required,name='dispatch')
 class AddMappingRuleFormView(FormView):
     form_class = AddMappingRuleForm
-    template_name = 'mapping/mappingrule_form.html'
+    template_name = "mapping/mappingrule_form.html"
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
 
-        scan_report_field = ScanReportField.objects.get(
-            pk=self.kwargs.get('pk')
-        )
+        scan_report_field = ScanReportField.objects.get(pk=self.kwargs.get("pk"))
```

Comment:
Just a quick comment, there seems to be a lot of changes like this
* `'` --> `"` 
* multi-line functions moved to single-line

Not sure if it's you making these changes, or it's come from a bad merge to the master where these changes are being overwritten.

The PEP8 style guide says to use the multiline split, make it easier to read
i.e.
```
 scan_report_field = ScanReportField.objects.get(
            pk=self.kwargs.get('pk')
        )
```


Now this is super pedantic of me to impose PEP8 style guides 😂  - im just wondering why these style changes are coming in with this MR? And making sure it's not something that's happening because of a merge to master, where someone else has made these changes?

Reply:
I _think_ it might have been a botched run of Python Black in VSCode.

Reply:
For example, Python black formatted the import models call like:

```
from .models import (
    ScanReport,
    ScanReportTable,
    ScanReportField,
    ScanReportValue,
    DataDictionary,
)

```

Is this in line with PEP8?